### DISCUSSION TITLE
How to run NPM scripts without issues when they don't exist

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#627] [Comments]
# How to run NPM scripts without issues when they don't exist

Discover how to seamlessly handle optional npm scripts in various build environments using the --if-present flag. Avoid errors and streamline CI processes.

https://kollitsch.dev/blog/2023/how-to-run-npm-scripts-without-issues-when-they-dont-exist/

**Comment by brycewray** [comment]
Needed this info about two days ago. :-) Thanks!
